src/rules/function-no-unknown/README.md

```diff
@@ -0,0 +1,64 @@
+# function-no-unknown
+
+Disallow unknown functions. Should be used **instead of** Stylelint's [function-no-unknown](https://stylelint.io/user-guide/rules/list/at-rule-no-unknown).
```

Comment:
@ybiquitous if this is meant to be used instead of `function-no-unknown`, then shouldn't this rule be checking against normal CSS functions too?

Reply:
Actually, the current implementation in this pull request does it:

```js
const ruleToCheckAgainst = "function-no-unknown";

// ...

utils.checkAgainstRule({ ruleName: ruleToCheckAgainst, ... });
```

1. Check against normal CSS functions via `utils.checkAgainstRule()`
2. Check against SCSS functions via `if (!ignoreFunctionsAsSet.has(funcName)) { ... }`

Reply:
okay, I see.

In that case could you also add some kind of unit test to verify that there is a warning for CSS functions? Just to make sure that the rule is working correctly.

Reply:
I've pushed 42889c2. Are the following cases enough?

- `unknown()` case for CSS function
- `color.unknown()` case for SCSS function

```js
  reject: [
    {
      code: "a { color: unknown(1); }",
      message: messages.rejected("unknown"),
      line: 1,
      column: 12
    },
    {
      code: "a { color: color.unknown(#6b717f, $red: 15); }",
      message: messages.rejected("color.unknown"),
      line: 1,
      column: 12
    }
  ]
```

Reply:
Oh yeah I wasn't thinking properly when I wrote my comment, sorry. 

What I was thinking about was that you could add a test that some non-SCSS function from this list: https://github.com/niksy/css-functions-list/blob/master/index.json passes and does not give a warning. 

But anyway those reject tests are also good to have 👍

Reply:
I understand your intention well. 👍🏼 

I've pushed 5dde89e to add more *accept* cases.

Reply:
Very nice, thanks!